pyproject.toml

```diff
@@ -55,7 +55,7 @@ dependencies = [
     "flask-wtf>=1.1.0, <2.0",
     "func_timeout",
     "geopy",
-    "gunicorn>=21.2.0, <22.0; sys_platform != 'win32'",
+    "gunicorn>=22.0.0; sys_platform != 'win32'",
```

Comment:
@dpgaspar should we include an upper limit on `<23.0`?

Reply:
We may be moving away from always pinning the upper limit: https://github.com/apache/superset/pull/28144#discussion_r1572740500